src/Cli/Commands/InitOptions.cs

```diff
@@ -82,6 +84,11 @@ public class InitOptions : Options
         [Option("auth.provider", Default = "StaticWebApps", Required = false, HelpText = "Specify the Identity Provider.")]
         public string AuthenticationProvider { get; }
 
+        // Since the rest.request-body-strict option does not have a default value, it is required to specify a value for this option if it is
```

Comment:
`auth.provider` is a string option. There is no default value of this option rest.request-body-strict. Since rest.request-body-strict maps to an Enum `CliBoolean`, when this option is not present in the init command, it assumes the value corresponding to 0th index of the enum which is `CliBoolean.None`. The values `CliBoolean.None` and `CliBoolean.True` are both serialized as request-body-strict=true in the config file.

Reply:
`InitOptions.RestRequestBodyStrict` NEED NOT assume the value corresponding to 0th index of the enum if we explicitly set the `Default = CliBoolean.True` value on the option.